packages/web-components/src/theme/set-theme.stories.ts

```diff
@@ -0,0 +1,17 @@
+import { html } from '@microsoft/fast-element';
+import { teamsDarkTheme, teamsHighContrastTheme, teamsLightTheme, webDarkTheme, webLightTheme } from '@fluentui/tokens';
+
+import { renderComponent } from '../helpers.stories.js';
+import { setTheme } from './set-theme.js';
+
+export default {
+  title: 'Theme/SetTheme',
+};
+
+export const SetTheme = renderComponent(html`
+  <fluent-button @click="${() => setTheme(webLightTheme)}">webLightTheme</fluent-button>
+  <fluent-button @click="${() => setTheme(webDarkTheme)}">webDarkTheme</fluent-button>
+  <fluent-button @click="${() => setTheme(teamsLightTheme)}">teamsLightTheme</fluent-button>
+  <fluent-button @click="${() => setTheme(teamsDarkTheme)}">teamsDarkTheme</fluent-button>
+  <fluent-button @click="${() => setTheme(teamsHighContrastTheme)}">teamsHighContrastTheme</fluent-button>
```

Comment:
Let's remove references to HC theme as we push for relying on the platform values here.